openapi/nexus.json

```diff
@@ -13663,6 +13700,43 @@
           "port_settings_id"
         ]
       },
+      "SwitchPortBgpPeerConfigCreate": {
```

Comment:
The prefixed names make sense here -- it's the sorta thing we'd want from uniquely-named submodules, but we're just doing explicitly.